aim/api/resource.py

```diff
@@ -148,3 +148,86 @@ def is_down(self):
                   (self.id, self.heartbeat_timestamp))
         return timeutils.is_older_than(self.heartbeat_timestamp,
                                        cfg.CONF.aim.agent_down_time)
+
+
+class Subnet(AciResourceBase):
+    """Resource representing a Subnet in ACI.
+
+    Identity attributes: name of ACI tenant, name of bridge-domain and
+    IP-address & mask of the default gateway in CIDR format (that is
+    <gateway-address>/<prefix-len>). Helper function 'to_gw_ip_mask'
+    may be used to construct the IP-address & mask value.
+    """
+
+    identity_attributes = ['tenant_name', 'bd_name', 'gw_ip_mask']
+    other_attributes = ['scope',
+                        'display_name']
+
+    _aci_mo_name = 'fvSubnet'
+    _tree_parent = BridgeDomain
+
+    def __init__(self, **kwargs):
+        super(Subnet, self).__init__({}, **kwargs)
```

Comment:
Just for reference, every time we add a AIM resource we need to specify meaningful default values for all the attributes that will show up in APIC. This will help consistent conversion (also, every time we do a GET on a AIM object, DB default values will show up... Which might not be meaningful at all).